src/status_im/core.cljs

```diff
@@ -54,5 +59,6 @@
   (async-storage/get-item :screen-height #(reset! shell.state/screen-height %))
 
   (dev/setup)
+  (prn "hermesEnabled ->" (is-hermes))
```

Comment:
@siddarthkay, at this point the logger is already set up, so it's better if we use it, e.g. `log/info`. Printing this info is useful. Good idea!